task_router/views.py

```diff
@@ -50,7 +53,7 @@ def incoming_sms(request):
 @csrf_exempt
 def incoming_call(request):
     """ Returns TwiML instructions to Twilio's POST requests """
-    resp = twiml.Response()
+    resp = VoiceResponse()
     with resp.gather(numDigits=1, action="/call/enqueue", method="POST") as g:
         g.say("For Programmable SMS, press one. For Voice, press any other key.")
```

Comment:
This is will actually not do what it should.
The `with resp.gather` will return a `VoiceResponse` instead of a `Gather` which it did before.
so the `g.say` will not happen the text in the Gather verb but in the Response instead.